app/src/main/java/com/seay/game_of_thrones/model/GoTCharacter.java

```diff
@@ -0,0 +1,44 @@
+package com.seay.game_of_thrones.model;
+
+public class GoTCharacter {
+
+    private int id;
```

Comment:
Our team's convention is that DTOs should include the `@SerializedName("id)"` annotation above any field that is being sent down from the server.